code/game/machinery/computer/Operating.dm

```diff
@@ -82,9 +159,43 @@
 		return 1
 	if ((usr.contents.Find(src) || (in_range(src, usr) && istype(src.loc, /turf))) || (istype(usr, /mob/living/silicon)))
 		usr.set_machine(src)
+
+	if(href_list["verboseOn"])
+		verbose=1
+	if(href_list["verboseOff"])
+		verbose=0
+	if(href_list["healthOn"])
+		healthAnnounce=1
+	if(href_list["healthOff"])
+		healthAnnounce=0
+	if(href_list["critOn"])
+		crit=1
+	if(href_list["critOff"])
+		crit=0
+	if(href_list["oxy_adj"]!=0)
+		oxyAlarm=oxyAlarm+text2num(href_list["oxy_adj"])
+	if(href_list["choiceOn"])
+		choice=1
+	if(href_list["choiceOff"])
+		choice=0
 	return
 
 
 /obj/machinery/computer/operating/process()
-	if(..())
-		src.updateDialog()
+
+	if(src.table && src.table.check_victim())
+		if(verbose)
+			if(patientName!=src.table.victim.name)
+				patientName=src.table.victim.name
+				atom_say("New patient detected, loading stats")
+				src.victim = src.table.victim
+				sleep(10)
+				atom_say("[src.victim.real_name], [src.victim.b_type] blood, [src.victim.stat ? "Non-Responsive" : "Awake"]")
+				atom_say("[round(src.victim.health)]")
+			if(src.victim.health <= -50 && crit)
+				playsound(src.loc, 'sound/machines/defib_success.ogg', 50, 0)
+			if(src.victim.getOxyLoss()>oxyAlarm)
+				playsound(src.loc, 'sound/machines/defib_saftyOff.ogg', 50, 0)
+			if(healthAnnounce)
+				atom_say("[round(src.victim.health)]")
+			sleep(60)
```

Comment:
what's the alternative? i just need the thing to wait so it doesn't spam the health every tick